tools/fog-local-network/fog_conformance_tests.py

```diff
@@ -414,17 +414,6 @@ def run(self):
         print(cmd)
         result = subprocess.check_output(cmd, shell=True)
 
-        # Report a missed block range from 0 to 1. This is needed after FOG-337 and should not be needed after FOG-393
-        cmd = ' '.join([
-            f'exec {FOG_PROJECT_DIR}/{target_dir(self.release)}/fog_ingest_client',
-            f'--uri insecure-fog-ingest://localhost:{BASE_INGEST_CLIENT_PORT}',
-            f'report-missed-block-range',
-            f'--start 0',
-            f'--end 1',
-        ])
-        print(cmd)
-        result = subprocess.check_output(cmd, shell=True)
```

Comment:
removing this is one of the main points of the PR

Reply:
Should be removed from the local network script as well I think